gems/smithy/spec/interfaces/client/endpoint_provider_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+describe 'Component: EndpointProvider' do
+  before(:all) do
+    @tmpdir = SpecHelper.generate(['Defaults'], :client, fixture: 'endpoints/default-values')
```

Comment:
Naming nit, EndpointDefaults (as opposed to model defaults which we may have to test later).

Reply:
The name here shouldn't matter right? It gets cleaned up so shouldn't be a conflict?

Reply:
It does get cleaned up. It was mostly just a naming of what the service is - it's endpoints related. I don't feel strongly.